src/trafgen/README

```diff
@@ -0,0 +1,8 @@
+trafgen docker archive belongs here.
```

Comment:
the git commit message only mentions gitignore, but I think the README part is more important. Proposition: "trafgen: document how to generate docker archive"
DO the same with the svtools commit.